Makefile

```diff
@@ -29,7 +29,7 @@ backend:
 
 frontend:
 	yarn install --pure-lockfile
-	GULP_ENV=prod yarn build
+	yarn encore production
```

Comment:
Btw, are we using this Makefile anywhere? Maybe we should include it in our pipeline, so it would be tested as well? 🤔

Reply:
@Zales0123 it is used in Zbyszek's Docker.